tests/test_storage.py

```diff
@@ -108,6 +109,20 @@ def make_remote_reply(source_uuid, journalist_uuid="testymctestface"):
     )
 
 
+def test_make_session_maker_updates_db_file_with_safe_perms():
+    """
+    When a db file already exists with unsafe permissions, ensure make_session_maker updates it to
+    the expected restricted 600 permissions.
+    """
+    with TemporaryDirectory() as temp_dir:
+        db_path = os.path.join(temp_dir, "svs.sqlite")
+        with open(db_path, "w"):
+            os.chmod(db_path, 0o100644)  # set unsafe perms
+            session_maker = db.make_session_maker(temp_dir)
```

Comment:
```suggestion
            db.make_session_maker(temp_dir)
```

The linting step failed because of the unused variable. (But the rest of the statement is what actions the code, so it's necessary.)

Other that that, yes, this fixes #1295 @creviera! :sparkles: